widget_driver/example/lib/widgets/coffee_consumption/coffee_counter/coffee_counter_header_section_driver.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:provider/provider.dart';
+import 'package:widget_driver/widget_driver.dart';
+
+import '../../../localization/localization.dart';
+
+part 'coffee_counter_header_section_driver.g.dart';
+
+@GenerateTestDriver()
+class CoffeeCounterHeaderSectionDriver extends WidgetDriver {
+  late Localization _localization;
+  late int _coffeeCount;
+
+  @override
+  void didUpdateBuildContext(BuildContext context) {
+    _localization = context.read<Localization>();
+    _coffeeCount = context.watch<int>();
```

Comment:
the `context.watch<int>()`  here now triggers this `didUpdateBuildContext` to be called every time the int changes from a parent widget..